Src/PerformanceCollector/Shared/Implementation/QuickPulse/QuickPulseEventSource.cs

```diff
@@ -138,6 +132,12 @@ public void SampleStoredEvent(int bufferLength, string applicationName = "dummy"
             this.WriteEvent(19, bufferLength, this.ApplicationName);
         }
 
+        [Event(7, Level = EventLevel.Verbose, Message = @"QuickPulseTelemetryModule has received a registration request from a QuickPulseTelemetryProcessor.")]
+        public void ProcessorRegistered(int count, string applicationName = "dummy")
+        {
+            this.WriteEvent(7, count, this.ApplicationName);
```

Comment:
Please change to int, sting, string so we do not have this issue again: https://github.com/Microsoft/ApplicationInsights-dotnet/issues/194